src/functionaltests/java/com/ericsson/gerrit/plugins/eiffel/linking/GitCommit.java

```diff
@@ -0,0 +1,7 @@
+package com.ericsson.gerrit.plugins.eiffel.linking;
+
+public class GitCommit {
```

Comment:
Moidel classes should maybe be in a seperate package.

Reply:
Its only used in the tests